src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -6,10 +6,13 @@
  */
 public class Bulldozer extends Machine {
```

Comment:
Don't begin class or method implementation with empty lines (lines 3-6). The class declaration should follow directly after the package statement.